engine/src/main/java/org/entando/entando/web/common/interceptor/ActivityStreamInterceptor.java

```diff
@@ -119,10 +118,15 @@ private String addParamsFromRequestBody(String prev, InputStream is) {
         try {
             CloseShieldInputStream cloned = new CloseShieldInputStream(is);
             String body = FileTextReader.getText(cloned);
-            byte[] bytes = body.getBytes();
-            ObjectMapper objectMapper = new ObjectMapper();
-            Map<String, String> params = objectMapper.readValue(bytes, HashMap.class);
-            return this.addParameters(prev, params);
+            int bodyLength = body.length();
```

Comment:
Could it be that the body is null?

Reply:
Hey @Kerruba ,
thanks for the review.
I did a code check and the variable body can't be null here.